specs/deployment-repo-update.md

```diff
@@ -0,0 +1,31 @@
+# Deployment Repo Update Automation
+
+## Summary
+
+The GitOps driven continuous deployment cycle starts with a change in the Git repository that contains resource manifests. The [Flux] provides the
+[Automated Image Update](https://docs.fluxcd.io/en/latest/references/automated-image-update.html) feature that continuously monitors the docker registry and automatically
+updates deployment repo when a new image is released. Obviously that functionality is not available for  Argo CD users.
```

Comment:
```suggestion
updates deployment repo when a new image is released. Obviously that functionality is not available for Argo CD users.
```

Reply:
s/Obviously that functionality/This functionality/

Reply:
fixed